website/source/api/secret/kmip/index.html.md

```diff
@@ -214,15 +214,16 @@ Delete a scope by name.
 - `scope` (`string: <required>`) - Name of scope. This is part of the request URL.
 - `force` (`bool: false`) - Force scope deletion. If KMIP managed objects have
   been created within the scope this param must be provided or the deletion will
-  fail.
+  fail. This value should be supplied as a query parameter, or as an argument in
+  the CLI.
 
 ### Sample Request
 
 ```
 $ curl \
     --header "X-Vault-Token: ..." \
-    --request POST \
-    https://127.0.0.1:8200/v1/kmip/scope/myscope/delete
```

Comment:
On the one hand I think it's good to provide an example with `force`.  On the other hand I think it's a bad idea to make the only delete example be with `force=true`, since it's liable to get copied and pasted without thinking by someone in a hurry.  You could provide two examples, or you could put the one example with `force=false` and thereby compel the user to have to manually edit it to be `true`.  That would be my preference.

Reply:
Agreed, I've updated the value to false now.